### DISCUSSION TITLE
how to concatenate queries

### DISCUSSION THREAD

**Discussion by deepajais95** [prisma/prisma#13416] [Q&A]
i have table student with column( name,department,id,company_id);
const studentDepartment: string | null = null;/// this value is coming from query param
cont  selectColumnName='name';/// this value is coming from query param

```
let query1 = `Select`;
query1=query1+ ` ${selectColumnName}`;
query1=query1+ ` from student`;
query1 = query1 + `WHERE company_id=${companyId}`;
if (studentDepartment) {
query1 = query1 + `AND department = ${studentDepartment}`;
}
await prisma.$queryRaw`
${Prisma.sql`${query1};`}`;
```

how to concantenate this query in prisma???

**Comment by nurul3101** [comment]
@deepajais95 👋 
What's the error that you are getting when executing the queryRaw?
Which `@prisma/client` are you running?

**Reply by vladi-stevanovic** [reply]
👋 Hello @deepajais95  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
